### ISSUE TITLE
Default Changelog Style changed from 2.4.0 to 2.5.0

### ISSUE THREAD

**Issue by HappyMeCoder** [9999years/latex-changelog#13]
I have changed my texlive installation from 2022 to 2024 and there was also an
update from changelog from v2.4.0 to v2.5.0. Since than the default style how it looks like has changed
and is completely different now. Same minimum example code was compiled.

v2.4.0  
![changelog_2_4_0](https://github.com/user-attachments/assets/38ad1cdc-9341-4ee8-80ef-503394f293cd)

v2.5.0  
![changelog_2_5_0](https://github.com/user-attachments/assets/275647fa-2a8a-4ea5-ae1f-bd3f7d004890)

Example Code:
I don't know why the code is not wrapped. I copy the example tex file i have used to the ZIP file  

```
\documentclass{scrreprt}
\usepackage{changelog}
%
\begin{document}
\tableofcontents
\chapter{History}   
\section*{Version 0.1}
%
\begin{changelog}
    \begin{version}[date=04.10.2018]
    \added
        \item First edition before first review
    \end{version}
\end{changelog}
%
\end{document} 
```

[ChangelogTest_2_4_0.pdf](https://github.com/user-attachments/files/17023787/ChangelogTest_2_4_0.pdf)
[ChangelogTest_2_5_0.pdf](https://github.com/user-attachments/files/17023788/ChangelogTest_2_5_0.pdf)
[ChangelogTest.zip](https://github.com/user-attachments/files/17023846/ChangelogTest.zip)

**Comment by 9999years** [comment]
Fixed in #14, I've submitted the new version (2.5.1) to CTAN, it should be published there in a few days. Not sure when it'll reach your TeX distribution.

**Comment by 9999years** [comment]
In the meantime, pasting this snippet after `\usepackage{changelog}` in your preamble (before `\begin{document}` should apply the fix:

```latex
\usepackage{etoolbox}
\ExplSyntaxOn
\cs_set:Npn \changelog__display_version
    {
        \tl_if_empty:NTF \g__changelog_version_tl
        {
            \tl_if_empty:NTF \g__changelog_date_tl
            {
                \GetTranslation{changelog-Unreleased}
                \tl_set:Nn \g__changelog_date_tl { \today }
            }
            {
                \tl_use:N \g__changelog_date_tl
            }
        }
        {
            \tl_use:N \g__changelog_version_tl
        }
    }

\patchcmd \changelog__short_version_item
    { \tl_use:N \g__changelog_version_tl } % Find this.
    { \changelog__display_version } % Replace with this.
    { } % Execute this on success.
    { } % Execute this on failure.

\cs_set:Npn \changelog__short_version_author_date
    {
        \mode_leave_vertical:
        \tl_use:N \g__changelog_author_tl
        \tl_if_empty:NF \g__changelog_date_tl
        {
            \tl_if_empty:NF \g__changelog_version_tl
            {
                \tl_if_empty:NF \g__changelog_author_tl
                { ~ }
                (\tl_use:N \g__changelog_date_tl)
            }
        }
    }

\cs_set:Npn \changelog__short_version_define_date { }
\ExplSyntaxOff
```
